back_end.py

```diff
@@ -0,0 +1,61 @@
+from flask import Flask
+from flask import request
+from flask import jsonify
+import json
+from mongo import Listing
+
+app = Flask(__name__)
```

Comment:
This is missing the CORS call from the Master branch & it's import. When merging, make sure to merge that portion of back_end.py from Master.

Reply:
Looking at the next three comments you left Jaxon here is a cohesive response to all:
Since these files were developed in the backend branch with a local database without connection to frontend connection I tried to keep from added/ importing things regarding the frontend to these files. When it comes to merging these files I will ensure that the current versions of them in the master branch will add the CORS in addition to changing
the database that get connected I will ensure that proper mongoclient is connected to and the proper collection is being accessed. Thank you for bringing this to my attention!